algo/src/test/java/com/hartwig/serve/sources/ckb/ActionableTrialFactoryTest.java

```diff
@@ -0,0 +1,128 @@
+package com.hartwig.serve.sources.ckb;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+import java.util.Set;
+
+import com.google.common.collect.Lists;
+import com.hartwig.serve.ckb.datamodel.CkbEntry;
+import com.hartwig.serve.ckb.datamodel.clinicaltrial.ClinicalTrial;
+import com.hartwig.serve.ckb.datamodel.clinicaltrial.ImmutableLocation;
+import com.hartwig.serve.ckb.datamodel.clinicaltrial.ImmutableVariantRequirementDetail;
+import com.hartwig.serve.ckb.datamodel.clinicaltrial.Location;
+import com.hartwig.serve.ckb.datamodel.clinicaltrial.VariantRequirementDetail;
+import com.hartwig.serve.datamodel.EvidenceDirection;
+import com.hartwig.serve.datamodel.EvidenceLevel;
+import com.hartwig.serve.datamodel.Knowledgebase;
+
+import org.apache.commons.compress.utils.Sets;
+import org.jetbrains.annotations.NotNull;
+import org.junit.Test;
+
+public class ActionableTrialFactoryTest {
+
+    @Test
+    public void canCreateActionableTrial() {
```

Comment:
I would rename this function to be more clear on what you are testing. Something like `canCreateActionableEntryForOpenTrialInAllowedCountry` or something similar.